src/opm/input/eclipse/Python/PythonInterp.cpp

```diff
@@ -82,6 +82,7 @@ PythonInterp::PythonInterp(bool enable) {
             throw std::logic_error("An instance of the Python interpreter is already running");
 
         this->guard = std::make_unique<py::scoped_interpreter>();
+        py::exec("import signal\nsignal.signal(signal.SIGINT, signal.SIG_DFL)");
```

Comment:
This is fine, of course, but we **might** consider using a raw string literal here to avoid the embedded `\n`.  E.g.,
```C++
py::exec(R"~~(import signal
signal.signal(signal.SIGINT, signal.SIG_DFL)~~)");
```

Reply:
I got fed up fighting python indentation crap, but i guess those tildes will help.

Reply:
> I got fed up fighting python indentation crap, but i guess those tildes will help.

Nah, the tildes are just to distinguish the closing parentheses of `signal.signal()` from that of the raw string literal itself.  In a raw string literal every space character is preserved so for Python syntax/indentation the `signal.signal()` statement must be at the very start of the line&mdash;i.e.,
```C++
            py::exec(R"~~(import signal
signal.signal(signal.SIGINT, signal.SEG_DFL)~~)");
```
For what it's worth, we have something similar in the implementation of [`OpmInputError::formatSingle()`](https://github.com/OPM/opm-common/blob/4b7cf09b752ea52ee6b4e71a4e79d736dbbfe442/src/opm/common/utility/OpmInputError.cpp#L63-L65).

Reply:
yeah, i just didn't like the broken indenting of the code. but i guess we just have to accept it for these things.